cli/command/container/stats_helpers_test.go

```diff
@@ -0,0 +1,49 @@
+package container
+
+import (
+	"testing"
+
+	"github.com/docker/docker/api/types"
+)
+
+func float64Equals(x, y float64) bool {
+	epsilon := 1e-9
+	return x-y < epsilon && y-x < epsilon
+}
+
+func TestCalculateMemUsageUnixNoCache(t *testing.T) {
+	// Given
+	stats := types.MemoryStats{Usage: 500, Stats: map[string]uint64{"cache": 400}}
+
+	// When
+	result := calculateMemUsageUnixNoCache(stats)
+	expected := float64(100.0)
+
+	// Then
+	if !float64Equals(result, expected) {
+		t.Errorf("mem = %f, want %f", result, expected)
+	}
```

Comment:
https://godoc.org/github.com/stretchr/testify/assert#InEpsilon

That way you don't have to create `float64Equals()`